build.gradle

```diff
@@ -27,7 +22,7 @@ buildscript {
         glide_version               = '4.8.0'
         gson_version                = '2.4'
 
-        kotlin_version              = '1.3.61'
+        kotlin_version              = '1.3.11'
```

Comment:
why do we need to downgrade kotlin version?

Reply:
Just reverting the changes from #41